osidb/models.py

```diff
@@ -1103,7 +1103,7 @@ def _validate_nonempty_impact(self):
         as the old flaws may have no impact
         """
         if not self.impact:
-            raise ValidationError("Impact value is required.")
+            self.alert("empty_impact", "Impact value is required.")
```

Comment:
I am not sure this is the correct approach. According to the conversation which triggered this task there are valid cases when the flaw has no impact. Then the alert is false positive and it will make people ignore it. As we do not have better solution now (should come as part of workflow rework) I would be for leaving the responsibility of providing the value on users and just not alert at all.

Reply:
Updated, please re-review.